I-D-Profile-Negotiation.xml

```diff
@@ -469,6 +475,11 @@ Content-Profile = "Content-Profile" ":"
 					Regardless of the outcomes of the negotiation process,
 					if the server knows that the representation conforms to one of more profiles,
 					it SHOULD list then in the "Content-Profile" header of the response.</t>
+				<t>The absence of an "Accept-Profile" header field in an HTTP response
+					SHOULD be taken as an indication
+					that the server does not support profiles for the given resource.
```

Comment:
\* does not support _profile-based content negotiation_?

Reply:
I'm not sure about this. "Does not support profiles": it likely does expect a specific profile, it is just not explicit about it.

Reply:
Agreed. "Does not support profile-based content-negotiation" is better.